packages/ERTP/package.json

```diff
@@ -40,7 +40,7 @@
     "@agoric/layer-cake": "^0.0.1",
     "@agoric/marshal": "^0.2.3",
     "@agoric/nat": "^2.0.1",
-    "@agoric/produce-promise": "^0.1.3",
+    "@agoric/promise-kit": "^0.1.3",
```

Comment:
With `@agoric/promise-kit` being a distinct package, I know I'll need to change all its version number stuff. I'll need some help with that.

Reply:
You can keep the same version.  Initial versions are completely arbitrary.